lib/pullrequest-decorator-reporter.js

```diff
@@ -47,6 +47,18 @@ const buildEmptyContext = () => ({
   list: [],
 });
 
+const decoratePullRequest = (githubService, report, options) => {
+  return Promise.allSettled([githubService.createPullRequestComment(report)
```

Comment:
I think you can write it like this : return Promise.allSettled([githubService.createPullRequestComment(report), githubService.createPullRequestCheck(report,options))
    .catch((exception) => console.error(`[+] An error occured : ${exception}`))
    .finally(() => {
      console.log(`[+] Finished collection: ${options.collectionName}`);
    })]

Reply:
I've written it like this, but I think that only catches the exception for the promise it proceeds, it wont catch any for the first element of the array?

Reply:
Done